src/main/java/com/openshift/internal/restclient/model/Pod.java

```diff
@@ -61,10 +74,38 @@ public String getHost() {
 		}
 		return images;
 	}
+	
+	/**
+	 * The logic of the method is a copied from 'podStatus' function of
+	 * [app/scripts/filters/resources.js] of [openshift/origin-web-console]
+	 */
 
 	@Override
 	public String getStatus() {
-		return asString(POD_STATUS);
+		if (get(POD_DELETION_TIMESTAMP).isDefined()) {
```

Comment:
@jcantrill i don't think we need to add it to `JbossDmrExtentions`, because `KubernetesResource` has `has(String key)` method itself, and we can check right from resources.
Anyway i'll change my `isDefined` to this `has`, as it really creates the child node. Didn't know about it